go.mod

```diff
@@ -78,6 +78,7 @@ require (
 	google.golang.org/grpc v1.27.1
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/yaml.v2 v2.4.0
+	honnef.co/go/tools v0.2.0 // indirect
```

Comment:
where it comes from, I did not see any new imports

Reply:
`Make check`？